src/render/attrs/index.js

```diff
@@ -10,6 +10,42 @@ const attrsNotToRender = {
   dangerouslySetInnerHTML: true
 };
 
+const attrsWithExplicitBoolValue = {
+  "aria-expanded": true,
+  "aria-haspopup": true
+};
```

Comment:
Just a thought here: there are many MANY properties and many cases. Maintaining that is a huge amount of work and it is very prone to errors. this is a potential major source of issues when comparing the react-dom version and rapscallion.

Wouldn't it be more reasonable to rely on the already crafted react internal accessible in the shared folder?

I was thinking we could maybe just have a custom function that would work a bit like https://github.com/facebook/react/blob/b1768b5a48d1f82e4ef4150e0036c5f846d3758a/src/renderers/dom/shared/DOMPropertyOperations.js#L138 that treats all the special cases handled by React currently and ensures that there is no difference when it comes to mutating arguments.